pom.xml

```diff
@@ -112,6 +122,7 @@
         <configuration>
           <instructions>
             <Bundle-SymbolicName>com.datastax.oss.driver.shaded.guava</Bundle-SymbolicName>
+            <Private-Package>com.google.common.primitives;-split-package:=merge-first</Private-Package>
```

Comment:
I wasn't expecting to see this again, @adutra , but I got warnings from the Maven bundler plugin indicating that we were back in a split package world when I just did the build naively.  I _suspect_ this is because of the overlapping class files mentioned [here](https://github.com/datastax/java-driver-shaded-guava/pull/4#issuecomment-638027201)... they might be landing us back in a split package world again.  That would also explain why I had to specify the split package based on unshaded rather than shaded package names.

I'll double-check to see if this changes any with your recent OSGi process fixes.

Reply:
Nevermind, I see you already removed it.  It makes sense that this wouldn't be necessary anymore; your compile/jar/shade/bundle strategy means that everything should be in a common class by the time we get to bundling (if I have the new process right) which removes the split-package concern that was present in my original impl.

Reply:
Correct. My intuition is that the shaded guava jar wasn't being generated correctly and was including the _non-shaded_ Guava substitution classes. With my changes in commit 
c09c517 I haven't seen any more problems.